src/platform_linux/laio.c

```diff
@@ -287,7 +291,11 @@ io_handle_deinit(laio_handle *io)
    }
    platform_assert(status == 0);
 
-   platform_free(io->heap_id, io->req);
+   platform_memfrag  memfrag = {.addr = io->req, .size = io->req_size};
```

Comment:
NOTE to myself: This should go away. Currently, `memfrag_init_size()` is a #define and this struct is exposed in `platform.h`. Rework this so the fields are hidden, and only `memfrag_init_size()` is exposed to client code.

This will prevent such naked assignments.